compiler/src/main/kotlin/io/spine/protodata/backend/CompilerEvents.kt

```diff
@@ -46,12 +46,14 @@ public object CompilerEvents {
      *
      * The resulting sequence is always finite, it's limited by the type set.
      */
-    public fun parse(request: CodeGeneratorRequest): Sequence<EventMessage> {
+    public fun parse(request: PluginProtos.CodeGeneratorRequest): Sequence<EventMessage> {
```

Comment:
I am not sure why we now include the reference to the parent type. Please review.